docs/source/modules/faker_demo.yaml

```diff
@@ -0,0 +1,113 @@
+   description:
+      name: Sample HPC specification
+      description: demo batch system and multiple worker configs for HPC workflows
+
+   env:
+     variables:
+       OUTPUT_PATH: ./name_studies
+       COLLECT: $(SPECROOT)/sample_collector.py
+       POST_PROC: $(SPECROOT)/sample_processor.py
+       PYTHON: /usr/WS2/white242/merlin_dev_2/venv_merlin_py3_7/bin/python
```

Comment:
I'd rather avoid making users change this path manually. If their venv is already activated won't it already be pointing to the proper python?

Reply:
Especially for cases when users pip-install merlin and don't even have a venv

Reply:
When i ran it the first iteration works, and the sample generation step found the venv alright, but subsequent calls seemed to be hitting a different python.  Still need to figure out what's going on there as the python scripts in the post process step couldn't find pandas on iteration 2 either, even though it's installed in the system pythons as well.